migrator/postgreshelper/load.go

```diff
@@ -71,7 +71,7 @@ func Load(conf *config.Config, databaseName string) (*pgxpool.Pool, *gorm.DB, er
 					return err
 				}
 			}
-			log.WriteToStderrf("connect to gorm: %v", gormSource)
+
```

Comment:
It's useful to have the log statement though. At least keep the log line with the host and db name, but ideally it can be printed as-is with the password redacted.